src/components/data/DataTable/DataTable.tsx

```diff
@@ -114,12 +114,81 @@ type DataTableState = {
  * DataTable component enables fetching data using predefined function and showing tabular data in a simple manner.
  *
  * ## Features
- * - data pagination
+ * - data fetch with pagination
  * - selectable rows
  * - expandable rows
  * - data sorting by columns
  * - filtering data
  *
+ * ## Data Fetch Details
+ * Fetch data is a function that accepts gridParam parameters
+ *
+ *
+ *      gridParams: {
+ *          _search: string;
+ *          currentPage: number;
+ *          pageSize: number;
+ *          sortColumn: string;
+ *          sortAscending: boolean;
+ *      };
+ *
+ *- `_search` is the search string that is input into search bar, empty string if not searchable or otherwise
```

Comment:
Maybe to show clear relation to `searchable` prop we could have:
```suggestion
 *- `_search` - if `searchable` prop is set to true, then it's a text provided in the search bar, it's an empty string otherwise
```
?